GitHub_Learning/Feature/Onboarding/Injection/OnboardingAssembly.swift

```diff
@@ -13,7 +13,13 @@ import Swinject
 final class OnboardingAssembly: Assembly {
     func assemble(container: Container) {
         container.autoregister(OnboardingViewController.self, initializer: OnboardingViewController.init)
+            .implements(OnboardingView.self)
         container.autoregister(OnboardingController.self, initializer: OnboardingControllerImpl.init)
             .inObjectScope(.container)
+        container.autoregister(UIWindow.self, initializer: UIWindow.init)
+        container.autoregister(OnboardingPresenter.self, initializer: OnboardingPresenterImpl.init)
+            .initCompleted { (r, presenter) in
+                (presenter as? OnboardingPresenterImpl)?.view = r.resolve(OnboardingView.self)
+        }
```

Comment:
Spatne odsazeni

Reply:
tak uplně nevim, jak zkusil jsem to okoukat z jiných CISu, tak snad myslíš tak jak jsem to udělal níž

Reply:
Takle jsem to nemyslel :D stacilo pridat jedno odsazeni. U ostatnich to nech. V principu jde o to ze ukoncovaci zavorka ma mit stejny odsazeni jako radek, na kterem je oteviraci.